stepper-2.0.md

```diff
@@ -184,6 +181,33 @@ using Stepper 2.0's custom float format described below.
 8  END_SYSEX                               (0xF7)
 ```
 
+**Stepper set maxSpeed**
+
+Sets the maximum speed in steps per second
```

Comment:
I think consistency should rule here. Besides, accelStepper's ```setMaxSpeed``` receives a float so it's not like we're making a performance improvement by using an integer.